package.json

```diff
@@ -105,5 +105,8 @@
     "terra-dev-site": "^3.0.0",
     "terra-toolkit": "^4.16.0",
     "xfc": "^1.2.1"
+  },
+  "dependencies": {
+    "keycode-js": "^1.0.4"
```

Comment:
This shouldn't be included at the root level

Reply:
You are correct :) Forgot to remove it the 2nd time. Ran into an issue where npm installing keycode-js was deleting the terra-list package. Made no sense, but forgot to remove it the 2nd time around.